index.html

```diff
@@ -329,6 +329,64 @@ <h3>Create better cloud outcomes with hands-on experience</h3>
             </div>
         </div>
     </section>
+    <!--keep uptodate section-->
+    <section class="keep-up-to-date-section">
+
+        <div class="keep-up-to-date-heading">
+            <h6 class="title-pink">KEEP UP TO DATE</h6>
+            <h3 class="title-sub">Discover more about Pluralsight</h3>
+        </div>
+        <div class="social-items-all">
+            <div class="social-item">
+                <div>
```

Comment:
Useless div

Reply:
those two divs , one for the image .and then for wrapping h and p tags